v1/message.go

```diff
@@ -37,6 +37,8 @@ type Content struct {
 	ChatID     string `json:"chat_id"`
 	Text       string `json:"text"`
 	ResponseTo string `json:"response-to"`
+	// The ENS name of the sender
+	Name string `json:"name"` // the ENS name of the sender
```

Comment:
You duplicated the comment and the formatting is still wrong.

Reply:
I have applied the automatic suggestion :) will fix that